### ISSUE TITLE
Attempt to split notebook fails

### ISSUE THREAD

**Issue by pbouffard** [fonsp/Pluto.jl#549]
I wanted to refactor a notebook into several smaller ones due to slow startup time of the combined notebook (largely due to several `@benchmark` calls). To do this I copied the file, opened copy in a text editor and deleted lines below the ones I wanted to keep.

Trying to open this new open I got this error:

```
KeyError: key UUID("96f7a8de-0980-11eb-301f-cfc1646b80a5") not found
Stacktrace:
 [1] getindex at ./dict.jl:467 [inlined]
 [2] load_notebook_nobackup(::IOStream, ::String) at /Users/patrick/.julia/packages/Pluto/UWKt3/src/notebook/Notebook.jl:158
 [3] #21 at /Users/patrick/.julia/packages/Pluto/UWKt3/src/notebook/Notebook.jl:170 [inlined]
 [4] open(::Pluto.var"#21#22"{String}, ::String, ::Vararg{String,N} where N; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at ./io.jl:325
 [5] open at ./io.jl:323 [inlined]
 [6] load_notebook_nobackup(::String) at /Users/patrick/.julia/packages/Pluto/UWKt3/src/notebook/Notebook.jl:169
 [7] load_notebook(::String, ::Bool) at /Users/patrick/.julia/packages/Pluto/UWKt3/src/notebook/Notebook.jl:186
 [8] open(::Pluto.ServerSession, ::String; run_async::Bool, compiler_options::Nothing) at /Users/patrick/.julia/packages/Pluto/UWKt3/src/webserver/SessionActions.jl:21
 [9] open at /Users/patrick/.julia/packages/Pluto/UWKt3/src/webserver/SessionActions.jl:15 [inlined]
 [10] (::Pluto.var"#try_launch_notebook_response#113"{Pluto.var"#try_launch_notebook_response#94#114"{Pluto.ServerSession}})(::typeof(Pluto.SessionActions.open), ::String; title::String, advice::String, home_url::String) at /Users/patrick/.julia/packages/Pluto/UWKt3/src/webserver/Static.jl:137
 [11] (::Pluto.var"#96#116")(::HTTP.Messages.Request) at /Users/patrick/.julia/packages/Pluto/UWKt3/src/webserver/Static.jl:166
 [12] (::Pluto.var"#84#102"{Bool,Pluto.var"#96#116",Pluto.ServerSession,Pluto.var"#add_set_secret_cookie!#99"{Pluto.ServerSession}})(::HTTP.Messages.Request) at /Users/patrick/.julia/packages/Pluto/UWKt3/src/webserver/Static.jl:101
 [13] handle(::HTTP.Handlers.RequestHandlerFunction{Pluto.var"#84#102"{Bool,Pluto.var"#96#116",Pluto.ServerSession,Pluto.var"#add_set_secret_cookie!#99"{Pluto.ServerSession}}}, ::HTTP.Messages.Request) at /Users/patrick/.julia/packages/HTTP/IAI92/src/Handlers.jl:253
 [14] handle(::HTTP.Handlers.Router{Symbol("##253")}, ::HTTP.Messages.Request) at /Users/patrick/.julia/packages/HTTP/IAI92/src/Handlers.jl:467
 [15] (::Pluto.var"#207#214"{Pluto.ServerSession,HTTP.Handlers.Router{Symbol("##253")},Base.RefValue{Function}})(::HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}) at /Users/patrick/.julia/packages/Pluto/UWKt3/src/webserver/WebServer.jl:210
 [16] handle at /Users/patrick/.julia/packages/HTTP/IAI92/src/Handlers.jl:269 [inlined]
 [17] #4 at /Users/patrick/.julia/packages/HTTP/IAI92/src/Handlers.jl:345 [inlined]
 [18] macro expansion at /Users/patrick/.julia/packages/HTTP/IAI92/src/Servers.jl:367 [inlined]
 [19] (::HTTP.Servers.var"#13#14"{HTTP.Handlers.var"#4#5"{HTTP.Handlers.StreamHandlerFunction{Pluto.var"#207#214"{Pluto.ServerSession,HTTP.Handlers.Router{Symbol("##253")},Base.RefValue{Function}}}},HTTP.ConnectionPool.Transaction{Sockets.TCPSocket},HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}})() at ./task.jl:356
```

As would make sense, the UUID that it errors on is that of the first cell, so this must be failing on looking up that cell in the cell order list which I clobbered. Note that copying the "Cell order" section from the bottom of the initial notebook file doesn't help; in that case it fails on the first UUID in the cell order list that isn't in the notebook.

So probably this is expected/as designed behavior, but if so the error message should probably say something more like "corrupted notebook" and not say to report the error, or (better) automatically "repair" the notebook by regenerating the cell order section of the file. Even better of course would be some sort of built-in refactoring tooling that would allow a subset of cells to be moved to a new notebook.

That aside, what would the recommended method be for doing what I describe? I suppose copying the file and then removing cells in the copy within Pluto should work but could be tedious if the refactoring involves splitting into several notebooks.

As I mentioned the motivation for splitting the notebook up in the first place was slowness in recomputing cells; if it were possible to avoid an automatic recalculation when a notebook is opened (without every cell's output showing an ugly `InterruptException` traceback) then this would lessen this motivation.

**Comment by fonsp** [comment]
Support for editing the notebook file using a different text editor is not within the scope of our project yet. But you point out an important missing feature: #259

**Comment by fonsp** [comment]
This specific issue was fixed by #1772, but this is still true:

> Support for editing the notebook file using a different text editor is not within the scope of our project yet.
